Bottom: c81efa5a70cf652856388e565e1c3c7fb452bcd0
Top:    cb9a06ee0947891fb744441360cf92ea9007ff38
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-08-17 16:22:32 -0400

Refresh of pwm

---

diff --git a/drivers/pwm/pwm-sunxi.c b/drivers/pwm/pwm-sunxi.c
index 52f9c95..b0d71ce 100644
--- a/drivers/pwm/pwm-sunxi.c
+++ b/drivers/pwm/pwm-sunxi.c
@@ -346,18 +346,6 @@ static int sunxi_pwm_enable(struct pwm_chip *chip, struct pwm_device *pwm)
 	}
 	if ((priv->revision == SUN5I) || (priv->revision == SUN7I))
 		ret = sunxi_pwm_busy(priv);
-
-{
-	int reg_val;
-	regmap_read(priv->regmap, SUNXI_PWM_CTRL_REG, &reg_val);
-	printk("JDS - SUNXI_PWM_CTRL_REG %08x\n", reg_val);
-	regmap_read(priv->regmap, SUNXI_PWM_CH0_PERIOD, &reg_val);
-	printk("JDS - SUNXI_PWM_CH0_PERIOD %08x\n", reg_val);
-	regmap_read(priv->regmap, SUNXI_PWM_CH1_PERIOD, &reg_val);
-	printk("JDS - SUNXI_PWM_CH1_PERIOD %08x\n", reg_val);
-	
-}
-
 exit:
 	mutex_unlock(&priv->lock);
 	return ret;
